deployment/datastore/address_ref.go

```diff
@@ -0,0 +1,61 @@
+package datastore
+
+import (
+	"errors"
+	"maps"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+var (
+	ErrAddressRefNotFound = errors.New("no such address ref can be found for the provided key")
+	ErrAddressRefExists   = errors.New("a address ref with the supplied key already exists")
```

Comment:
```suggestion
	ErrAddressRefExists   = errors.New("an address ref with the supplied key already exists")
```
or just remove `a` ?